### ISSUE TITLE
Cannot run the program (on mac)

### ISSUE THREAD

**Issue by tvillingett** [hobbyquaker/homekit2mqtt#1]
Hi,
when I try to install it, after
```
sudo npm install -g homekit2mqtt
```

I try to start it with
```
Anders-Erikssons-Mac-mini:.homebridge anders$ homekit2mqtt
/usr/local/bin/homekit2mqtt: line 1: syntax error near unexpected token `('
/usr/local/bin/homekit2mqtt: line 1: `var pkg = require('./package.json');'
```

I guess it is something simple but I cannot get it...

**Comment by hobbyquaker** [comment]
Oh, I guess this is my fault. Please try `node /usr/local/lib/node_modules/homekit2mqtt/index.js --help` this should work. Will fix the link in /usr/local/bin asap.

**Comment by tvillingett** [comment]
ok, thanks!

**Comment by hobbyquaker** [comment]
should be fixed in 0.3.0
